PlayCover/en.lproj/Localizable.strings

```diff
@@ -97,6 +97,8 @@
 "alert.install.injectPlayToolsQuestion" = "Do you want to inject PlayTools into this application?";
 "alert.install.playToolsInformative" = "PlayTools is a tool that allows you to bind keys to screen touches, custom display resolutions, and more, but it may break certain apps";
 "alert.supression" = "You can change this in the PlayCover preferences";
+"alert.differentBundleIdKeymap.message" = "The name of the keymap is different from the bundle ID of the app!";
```

Comment:
I don't think most people will understand what bundle IDs are, so it would be better if `alert.differentBundleIdKeymap.text` was `alert.differentBundleIdKeymap.message` and have `alert.differentBundleIdKeymap.text` be something along the lines of "Are you sure you want to import this keymap?" so that the `Proceed` button makes more sense.